Umbraco-Cloud/Release-Notes/2022-08-19-ReleaseNotes.md

```diff
@@ -0,0 +1,30 @@
+# Release Notes, August 19, 2022
+
+_Dedicated Resources - Projects Dashboard (beta)_
+
+## Key Takeaways
+
+- **Dedicated Resources** - A new feature offering dedicated resources for your Umbraco Cloud project has been released. You can upgrade your projects to use dedicated resources with the click of a button.
+- **Projects Dashboard (beta)** - The beta version of the Projects Dashboard offers the first sneak preview of the future projects overview in the Umbraco Cloud Portal.
+
+## [Dedicated Resources](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/Dedicated-Resources/)
+
+With Dedicated Resources for Umbraco Cloud, you can now choose when to **scale up and down on the resources available for a project**. On top of that, the project is moved from shared to **dedicated hosting**. 
+
+You can access Dedicated Resources from the Settings menu on your Cloud project. Here you’ll find 3 different dedicated options (CPU and memory configurations) to choose from bringing flexibility in how requirements are met, and how much extra compute power is added. As soon as you select and confirm an option the process of migrating to dedicated resources begins. If the extra resources are no longer needed, the project can be moved back to shared resources in the same way.
+
+![ProjectsDashboard](images/DedicatedResourceMvp.gif)
+
+Dedicated Resources is currntly available for projects on **Professional** and **Enterprise** plans. More options will be added later in the year to make the feature available for Starter and Standard plans as well. 
+
+## Projects Dashboard (beta)
+
+We are currently working on multiple initiatives to give Umbraco Cloud users a fantastic **developer-friendly experience**. One of these is the Projects Dashboard (as it’s known internally). Together with partners and Cloud users, we have worked to identify what would be the most useful features and tools to provide an overview of all your projects.
```

Comment:
> One of these is the Projects Dashboard (as it’s known internally).

Is the "as it's known internally" necessary?
Is it because the name might change later, because then perhaps we could say something like "this is what it's currently called".

What do you think?

Reply:
Yup, your suggestion would be better. You could also skip the content in the parenthesis. That would be find with me.

Reply:
Let's skip it then.
I'll amend and merge 💪